src/test/Stack/ArgsSpec.hs

```diff
@@ -72,19 +91,28 @@ interpreterArgsSpec =
 
       checkLines args = forM_
         (interpreterGenValid lineComment args)
-        (testAndCheck acceptSuccess args)
+        (testAndCheck (acceptSuccess False) args)
```

Comment:
I suspect it would be more legible to have an inline `data Literacy = Literate | Illiterate` here, rather than an unmarked `Bool` (or maybe a `newtype` on Bool?) Do you agree?